### PR TITLE
Unfiltered people velocity tracker

### PR DESCRIPTION
**Pull request by ablasdel**
Indigo stuff:
specify publisher queue_size

PEP8 (missed some in the previous)

Velocity tracker:
I couldn't get the filtered tracker to function so I created an unfiltered tracker.

### CODE REVIEW

**Comment by ablasdel** [comment]
This pull provides a workaround for https://github.com/wg-perception/people/issues/24

It basically just allows running the tracker without filtering because the filtering doesn't seem to be working at the moment.

File: people_velocity_tracker/scripts/tracker.py [hunk 1]
```diff
@@ -91,9 +91,14 @@ def __init__(self):
...
         self.TIMEOUT = rospy.Duration(rospy.get_param('~timeout', 1.0))
         self.sub = rospy.Subscriber('/people_tracker_measurements',
-                                    PositionMeasurementArray, self.pm_cb)
-        self.mpub = rospy.Publisher('/visualization_marker', Marker)
-        self.ppub = rospy.Publisher('/people', People)
+                                    PositionMeasurementArray,
+                                    self.pm_cb)
+        self.mpub = rospy.Publisher('/visualization_marker',
+                                    Marker,
+                                    queue_size=0)
```
Comment (mikeferguson) [inline people_velocity_tracker/scripts/tracker.py]: queue_size = 0 is generally a bad idea -- see http://wiki.ros.org/rospy/Overview/Publishers%20and%20Subscribers#Choosing_a_good_queue_size

I would suggest something like 10 for this message

Comment (ablasdel) [inline people_velocity_tracker/scripts/tracker.py]: You are correct! I had meant to call out the default behavior was being executed (e.g. Queuesize=None) I mistook zero and None to have the same behavior... 

Sorry for the mistake! I'll fix it.

**Comment by ablasdel** [comment]
@DLu & @mikeferguson This pull should be safe to merge now.

**Pull request merged**